Competition2017/tests/competition/subsystems/servo/SetServoCommandTest.java

```diff
@@ -0,0 +1,22 @@
+package competition.subsystems.servo;
+
+import org.junit.Test;
+
+import competition.BaseTest;
+import edu.wpi.first.wpilibj.MockServo;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class SetServoCommandTest extends BaseTest {
+    
+    @Test
+    public void test(){
+        SetServoCommand command = injector.getInstance(SetServoCommand.class);
+        ServoSubsystem subsystem = injector.getInstance(ServoSubsystem.class);
+        command.setTargetPose(0.4);
+        command.execute();
```

Comment:
generally a good idea to call initialize on your command here first in case it explodes/changes things